templates/zerver/hello.html

```diff
@@ -42,7 +42,6 @@ <h1>Chat for distributed teams.</h1>
                         <div class="item-container">
                             <div class="item active">
                                 <div class="item-inner">
-
```

Comment:
Why did you delete the empty line here?

Reply:
I didn't edit anything near line 45, but I did run prettier, so that could have been why an empty line was removed. I hope no code was lost?

Reply:
I ran this command -

`./tools/lint --fix
`
As it had failed in a test earlier.

Reply:
Yeah, I guess then it's alright.

Reply:
> Now, Everything LGTM.

Yay!!